### PR TITLE
feat: link to external event documentation added

### PR DESCRIPTION
**Pull request by thim81**
## Motivation

A PR to introduce the capability to add 1 or multiple external links.
See https://github.com/boyney123/eventcatalog/issues/55 for the details on this proposal.

### Have you read the [Contributing Guidelines on pull requests](https://github.com/boyney123/eventcatalog/blob/main/CONTRIBUTING.md#pull-requests)?

Yes

### CODE REVIEW

**Comment by thim81** [comment]
@boyney123 

Some screenshots:
2022-01-18 at 11 19 30
2022-01-18 at 11 20 02@2x

Todo:
- Review styling
- Update docs on the new frontMatter property `extLinks`

**Comment by thim81** [comment]
Request: In case this PR gets approved, please consider a "Squash & merge" since the Git history is embarrassing 🤦

File: examples/basic/events/AddedItemToCart/index.md [hunk 1]
```diff
@@ -12,6 +12,10 @@ domains:
 owners:
     - dboyne
     - mSmith
+extLinks:
```
Comment (boyney123) [inline examples/basic/events/AddedItemToCart/index.md:15]: @thim81 what do you think of having `externalLinks`, might be slightly more descriptive , `extLinks` I think might require some lookup or understanding? What do you think?

**Comment by boyney123** [comment]
Thank you @thim81 for the contribution! 🙏, I love the idea!


**Quick on on the icon:**

@thim81 on hero icons, they have an icon for external links (which I believe is standard, I've seen elsewhere).

image

Maybe on the right of the text? 

image

File: examples/basic/events/AddedItemToCart/index.md (hunk 1 above)
Comment (thim81) [reply examples/basic/events/AddedItemToCart/index.md:15]: Fully right, I'll update all the changes to use externalLinks

**Comment by thim81** [comment]
I would keep the icon on the left, to keep it consistent with the others button styles.

**Comment by thim81** [comment]
I'll add the new `externalLinks` property to docs

**Comment by boyney123** [comment]
Ok great, looking good to me @thim81 , any chance you could update the docs in the PR?

https://eventcatalog.dev/docs/api/event-frontmatter

Add docs explaining the new field and how to use it 🙏?

**Comment by thim81** [comment]
I just added the documentation.

**Pull request merged**